app/screens/find_channels/filtered_list/user_item/user_item.tsx

```diff
@@ -52,17 +52,12 @@ const UserItem = ({currentUserId, onPress, teammateDisplayNameSetting, testID, u
     const {formatMessage, locale} = useIntl();
     const theme = useTheme();
     const styles = getStyleSheet(theme);
-    let displayName = displayUsername(user, locale, teammateDisplayNameSetting);
+    const isOwnDirectMessage = currentUserId === user.id;
+    const displayName = displayUsername(user, locale, teammateDisplayNameSetting);
 
     const handleOnPress = useCallback(() => {
         onPress(user.id, displayName);
-    }, [locale, user, displayName]);
-
-    const isOwnDirectMessage = currentUserId === user.id;
-
-    if (isOwnDirectMessage) {
-        displayName = formatMessage({id: 'channel_header.directchannel.you', defaultMessage: '{displayName} (you)'}, {displayName});
-    }
+    }, [user.id, displayName]);
```

Comment:
Just to confirm, we consider `onPress` constant during the lifetime of the object?

Reply:
we do, but will add to the dependencies